File: Dik-Ucgenin-Alani-ve -Cevresi.py
#!/usr/bin/env python
# -*- coding: utf-8 -*-
#python3

def hesapla(k1, k2):
	alan = (k1*k2)/2
	cevre = pow(pow(k1, 2) + pow(k2, 2), 1/2) + k1+k2
	return "Üçgenin Alanı: {}\nÜçgenin Çevresi: {}".format(alan, cevre)

kenar1 = int(input("Üçgenin Birinci Dik Kenarını Giriniz: "))
kenar2 = int(input("Üçgenin İkinci Dik Kenarını Giriniz: "))

print(hesapla(kenar1, kenar2))

File: DikUcgeninAlaniveCevresi.cs
 class Program
    {
        static void Main(string[] args)
        {
            Console.WriteLine("Üçgenin birinci dik kenar uzunluğunu girin:");
            int Kenar1 = Convert.ToInt32(Console.ReadLine());

            Console.WriteLine("Üçgenin ikinci dik kenar uzunluğunu girin:");
            int Kenar2 = Convert.ToInt32(Console.ReadLine());

            int Hipotenus = (int)Math.Sqrt(Math.Pow(Kenar1, 2) + Math.Pow(Kenar2, 2));

            Console.WriteLine("Dik üçgenin Alanı= {0}", AlanHesapla(Kenar1, Kenar2));
            Console.WriteLine("Dik üçgenin çevresi = {0}", CevreHesapla(Kenar1, Kenar2, Hipotenus));
            Console.ReadKey();
        }

        static int AlanHesapla(int k1, int k2)
        {
            int Alan = (k1 * k2) / 2;
            return Alan;
        }
        static int CevreHesapla(int k1, int k2, int k3)
        {
            int Cevre = k1 + k2 + k3;
            return Cevre;
        }
    }
